v2/config/samples/eventgrid/v1alpha1api20200601_eventsubscription.yaml

```diff
@@ -6,12 +6,14 @@ metadata:
 spec:
   owner:
     name: sample-domain-topic
+    group: resources.azure.com
+    kind: ResourceGroup
   destination:
     storageQueue:
       endpointType: StorageQueue
       properties: 
         resourceReference: 
           group: storage.azure.com
           kind: StorageAccount
-          name: samplekubestorage
+          name: samplekubestoragealpha
```

Comment:
Have to leave the changed naming for refs, as they get deployed before the resources, so its just to avoid the conflict between alpha and beta resources

Reply:
Gotcha. That's something we can maybe iterate on in the tool (to generate ref names too)